recipes/apr-util/all/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+import os
```

Comment:
test_package failed to compile on my machine...
```
/opt/rh/devtoolset-8/root/usr/libexec/gcc/x86_64-redhat-linux/8/ld: cannot find -luuid
```
not sure though if this hsould be addressed here.